script.skinshortcuts/resources/docs/FAQ.md

```diff
@@ -0,0 +1,84 @@
+# End User FAQ
```

Comment:
please avoid submitting the docs folder. Add it to .gitattributes

Reply:
Not exactly sure how to do this...
```
/docs               export-ignore
```
Is that correct?

Reply:
Since no response I just deleted the docs folder